integration/tests/basic/erc/test_ERC1820.py

```diff
@@ -0,0 +1,70 @@
+from conftest import EnvName
+from utils.accounts import EthAccounts
+from utils.web3client import NeonChainWeb3Client
+
+
+class TestERC1820PseudoIntrospectionRegistryContract:
```

Comment:
please add @allure.feature and @allure.story and some description to the doc md file